CommunityFridgeMapApi/functions/dev/get_all/app.py

```diff
@@ -0,0 +1,70 @@
+import os
+import logging
+from botocore.exceptions import ClientError
+import json
+import sys
+try:
+    from db import get_ddb_connection
+except:
+    #If it gets here it's because we are performing a unit test. Here is an example of someone having a similar issue
+    #https://stackoverflow.com/questions/69592094/pytest-failing-in-aws-sam-project-due-to-modulenotfounderror
+    from dependencies.python.db import get_ddb_connection
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+class GetAllFridgesHandler:
```

Comment:
this file will eventually get deleted. It's purpose is to be able to test that a db query can be made successfully locally